docs/config-file-ort-yml.md

```diff
@@ -235,3 +235,33 @@ resolutions:
     reason: "LICENSE_ACQUIRED_EXCEPTION"
     comment: "Commercial Qt license for the project was purchased, for details see https://jira.example.com/issues/SOURCING-5678"
 ```
+
+### Resolving Vulnerabilities
+
+Vulnerabilities detected during the advisor step might want to be ignored if a vulnerability is a false positive, or 
```

Comment:
Line 241 till 244 can be deleted as the basic of resolutions are already explained above